### PR TITLE
HARP-7582 Move IBCT utils to /coresdk

### PR DESCRIPTION
**Pull request by arekspiewak1991**
Signed-off-by: Arkadiusz Spiewak <ext-arkadiusz.spiewak@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,14 @@
         "test/performance"
```
[the remark this answers is not in the document]
Comment (arekspiewak1991) [reply package.json]: done

File: @here/harp-test-utils/lib/rendering/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Rendering Test Utils
```
[the remark this answers is not in the document]
Comment (arekspiewak1991) [reply @here/harp-test-utils/lib/rendering/README.md]: remove... but will write the new one

**Comment by arekspiewak1991** [comment]
@heremaps-bot retest this please

**Comment by arekspiewak1991** [comment]
@heremaps-bot retest this please

**Comment by arekspiewak1991** [comment]
@heremaps-bot retest this please

File: @here/harp-test-utils/lib/rendering/FeedbackCli.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                }
+                someTestsFailed = true;
+            }
+        } else {
+            logger.log(`${referencePath}: success`);
+        }
+    });
+    return !someTestsFailed;
+}
+let baseResultsPath = "test/ibct-results";
```
Comment (zbigg) [inline @here/harp-test-utils/lib/rendering/FeedbackCli.ts]: `rendering-test-results`

File: @here/harp-test-utils/lib/rendering/ImageBasedTestHelper.ts [hunk 4]
```diff
@@ -0,0 +1,246 @@
+/*
```
Comment (zbigg) [inline @here/harp-test-utils/lib/rendering/ImageBasedTestHelper.ts]: Rename please to `RenderingTestHelper.ts` ... possibly in following PR

File: @here/harp-test-utils/lib/rendering/FeedbackServer.ts [hunk 5]
```diff
@@ -0,0 +1,199 @@
+/*
```
Comment (zbigg) [inline @here/harp-test-utils/lib/rendering/FeedbackServer.ts]: Rename to `RenderingTestResultServer`

File: @here/harp-test-utils/lib/rendering/FeedbackCli.ts [hunk 6]
```diff
@@ -0,0 +1,115 @@
+/*
```
Comment (zbigg) [inline @here/harp-test-utils/lib/rendering/FeedbackCli.ts]: RenderingTestResultCli

File: @here/harp-test-utils/lib/rendering/FeedbackCommon.ts [hunk 7]
```diff
@@ -0,0 +1,50 @@
+/*
```
Comment (zbigg) [inline @here/harp-test-utils/lib/rendering/FeedbackCommon.ts]: `RenderingTestResultCommon.ts`

File: @here/harp-test-utils/lib/rendering/ImageLocator.ts [hunk 8]
```diff
@@ -0,0 +1,60 @@
+/*
```
Comment (zbigg) [inline @here/harp-test-utils/lib/rendering/ImageLocator.ts]: `ReferenceImageLocator,ts`

**Pull request merged**